website_and_docs/content/documentation/selenium_manager.en.md

```diff
@@ -68,6 +68,68 @@ edge can only be installed in Windows with administrator permissions
 
 Therefore, administrator permissions are required to install Edge in Windows automatically through Selenium Manager, and Edge is eventually installed in the usual program files folder (e.g., `C:\Program Files (x86)\Microsoft\Edge`).
 
+## Example WebDriver Code
+
+These are provided in C# but similar code would work for other languages. 
```

Comment:
I really do not want to do single language examples. Things got bad when everything was java 😂

Reply:
Fair enough - it'll take me some time to cobble together other languages, but I'll probably be able to borrow some sample code

Reply:
I'm not sure any of the code examples are necessary. They are literally duplicating what we have in our getting started with Selenium section.